demos/demo_working_copy.sh

```diff
@@ -27,24 +26,27 @@ run_command "jj log"
 
 comment "Start working on a new change off of master:"
 run_command "jj co master"
-run_command "jj log"
-
-comment "Note that the working copy is now clean; the
+comment "Note that we were told the working copy is now empty (AKA clean). The
 \"goodbye\" change stayed in its own commit:"
+
+run_command "jj log"
+comment "Let's do a sanity check: 'jj status' should tell us that
+the working copy is clean."
 run_command "jj status"
 
 comment "Modify a file in this new change:"
 run_command "echo \"Hello everyone!\" > README"
+run_command "jj status"
 
 comment "The working copy is not special; we can, for
 example, set the description of any commit.
 First, set it on the working copy:"
-run_command "jj describe -m everyone"
+# The output with the description of the working copy messes up the parallel
+# a bit, so we redact it.
+run_command_output_redacted "jj describe -m everyone"
```

Comment:
I wonder if readers are instead going to wonder why the output is redacted. We'll see (no action requested)

Reply:
Along these lines, I made this text grey to make it less visible. Perhaps as a result fewer people will be nerd-sniped.